XLTF7267-BHA+: Document how to change the PLOAM password (#212)
hack-gpon/hack-gpon.github.io: _ont_xgs/ont-hisense-ltf7267-bha+.md

Reboot ONT to apply the change

## Setting ONU GPON PLOAM password

This ONT seems to be supporting a PLOAM password up to 288 bits in lenghth (36 ASCII characters, 72 Hex digits).

The PLOAM password is stored into 32 bit chunks (4 ASCII characters / 8 Hex digits), each byte swapped. 

So, starting from the following PLOAM in ASCII format

```
A1B2C3D4E5
```

It gets translated into the following HEX value:

```
0x41314232433344344535
```

Which is then split into the following blocks (the last block gets padded with 0 to reach 8 digits)

```
BLOCK 0: 0x41314232
BLOCK 1: 0x43334434
BLOCK 2: 0x45350000
```

> Each block is then byte swapped (i.e. with read each block of two digits from right to left)

Author: ```suggestion
Each block is then byte swapped (i.e. read each sequence of two digits from right to left)
```

```
BLOCK 0: 0x32423141
BLOCK 1: 0x34443343
BLOCK 2: 0x00003545
```

> And then you can finally save it by chaning the configuration file

Author: ```suggestion
And then you can finally persist it by changing the configuration file
```

```sh
# vi /config/scfg.txt
```

Append lines below to the file and save it to change the PLOAM password

```
INT             CFG_ID_PON_REGISTRATION_ID0                                         = 0x32423141;
INT             CFG_ID_PON_REGISTRATION_ID1                                         = 0x34443343;
INT             CFG_ID_PON_REGISTRATION_ID2                                         = 0x00003545;
```

> Reboot ONT to apply the change.

Author: ```suggestion
Reboot the ONT to apply the change.
```

Reboot ONT to apply the change

## Setting ONU GPON PLOAM password

> {% include alert.html content="The value 0x0 is null, take note of your LoID and password from the original ONT" color="red" %}

Reviewer: LoID and password ??
